### DISCUSSION TITLE
Support custom trackBy directives

### DISCUSSION THREAD

**Discussion by JamesIves** [angular-eslint/angular-eslint#941] [New Rule Proposals]
I've been working with the `use-track-by-function` rule and I was wondering if would be possible to support custom directives. For  example I wanted to [implement something like this](https://stackoverflow.com/questions/42108217/how-to-use-trackby-with-ngfor/68598194#68598194) into my code base but sadly it makes the linter unhappy and there doesn't seem to be any way to turn it off without using an ignore statement.

**Comment by JamesHenry** [comment]
`alias` was added here https://github.com/angular-eslint/angular-eslint/pull/1497
